internal/corazawaf/transaction_test.go

```diff
@@ -429,6 +430,28 @@ func TestAuditLog(t *testing.T) {
 	}
 }
 
+func TestParseAuditLog(t *testing.T) {
+	tx := makeTransaction(t)
```

Comment:
I wonder why we need to create a transaction to test the log parts.

Reply:
sorry,I just cody the TestAuditLog()